app/design/adminhtml/default/default/template/ebizmarts/mailchimp/customer/tab/mailchimp.phtml

```diff
@@ -4,7 +4,7 @@ $interest = $this->getInterest();
 <div class="fieldset-wrapper">
     <div class="fieldset-wrapper-title">
         <span class="title">
-            <?php /* @escapeNotVerified */ echo $this->quoteEscape($this->__('Mailchimp Information')) ?>
+            <?php /* @escapeNotVerified */ echo $this->gethelper()->mcEscapeQuote($this->__('Mailchimp Information')) ?>
```

Comment:
This can be more straight forward by calling only one function that calls the helper internally.